Backpack/Skeleton/Classes/BPKSkeletonSize.swift

```diff
@@ -0,0 +1,69 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import UIKit
+
+public enum BPKSkeletonSize: Equatable {
```

Comment:
Something for @frugoman, but I'm wondering if it would make sense to move this configuration to Backpack common, as both packages need this? (And we risk having different configs across UIKit / SwiftUI)

Reply:
This is a good idea, although not sure how to implement it correctly so far, so I think accepting this as is on this PR is ok. we can migrate it later as a task for Koala